src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -28,8 +28,10 @@
     private final int ARBITRARY_STUDENT_TOT_MADEUP_1 = 2;
     private final int ARBITRARY_STUDENT_TOBE_MADEUP_1 = 3;
     private final int ARBITRARY_STUDENT_EXCUSED_1 = 3;
+    private final int ARBITRARY_STUDENT_PRESENT_1 = 1;
     private final int ARBITRARY_STUDENT_MIN_MISSED_1 = 50;
     private final double ARBITRARY_STUDENT_PERCENT_1 = 2.5;
+    private final String ARBITRARY_SIS_USER_ID_1 = "SIS_ID_1";
     //Student 2 Data
     private final long ARBITRARY_STUDENT_2 = 222784;
     private final String ARBITRARY_STUDENT_NAME_2 = "Student, Dos";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ARBITRARY_STUDENT_NAME_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)